src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -18,6 +18,6 @@
 /**
  * @deprecated since Symfony 4.4, use Symfony\Component\ErrorCatcher\ErrorHandler instead.
  */
-class ErrorHandler extends BaseErrorHandler
+final class ErrorHandler extends BaseErrorHandler
```

Comment:
this class should be removed instead, it's deprecated (same for many other classes in the component)

Reply:
OK, I see what you mean. I'm getting lost with the 4.4 / 5.0 release 😄 I will do it in another MR then.